README.md

```diff
@@ -1,72 +1,20 @@
-![sidebase-logo-for-dark](.github/sidebase_logo_dark_icon_font_for_dark_bg_long.svg#gh-dark-mode-only#gh-dark-mode-only)
-![sidebase-logo-for-light](.github/sidebase_logo_light_icon_font_white_bg_long.svg#gh-light-mode-only)
-
-# sidebase
-
- <a href="https://github.com/sidebase/sidebase/actions?query=branch%3Amain+event%3Apush"><img src="https://github.com/nuxt/framework/workflows/ci/badge.svg?branch=main&event=push" alt="CI Status"></a>
- [![GitHub stars](https://badgen.net/github/stars/sidebase/sidebase)](https://GitHub.com/sidebase/sidebase/)
- [![License][license-src]][license-href]
-[![Follow us on Twitter](https://badgen.net/badge/icon/twitter?icon=twitter&label)](https://twitter.com/sidebase_io)
-[![Join our Discord](https://badgen.net/badge/icon/discord?icon=discord&label)](https://discord.gg/9MUHR8WT9B)
-
+# sidebase Nuxt3 starter
 
 sidebase is a modern, best-practice, batteries-included fullstack-app starter based on Nuxt 3 and TypeScript.
 
-With this nuxt 3 starter you get production-ready frontend + backend projects while still having fun! [Atinux, CEO of Nuxt](https://github.com/Atinux) said to sidebase on Twitter:
-> Beautiful work on sidebase!
-
 ## Quick start
 
-![sidebase-preview-for-dark](.github/preview/dark_mode.png#gh-dark-mode-only#gh-dark-mode-only)
-![sidebase-preview-for-light](.github/preview/light_mode.png#gh-light-mode-only)
-
-1. Use the official [`nuxi`-cli](https://v3.nuxtjs.org/api/commands/init) to start:
-    ```sh
-    npx nuxi@latest init -t community/sidebase
-    ```
-2. Go into the `nuxt-sidebase/` directory
```

Comment:
ohhh, is this also outdated in the main readme? If so, we should at least open an issue now to re-write this in the main readme (and resolve that soon? maybe schedule the work for next work??)